### ISSUE TITLE
Line break in Label

### ISSUE THREAD

**Issue by willwade** [asterics/AsTeRICS-Grid#322]
Hi 

I'm trying to replicate the attached book in asterics. The first page, I'm having difficulty as I can't line break in the label.. Or can I already do this?

[PDF file of phrase book only.pdf](https://github.com/asterics/AsTeRICS-Grid/files/12584839/PDF.file.of.phrase.book.only.pdf)

<img width="990" alt="Screenshot 2023-09-12 at 10 38 39" src="https://github.com/asterics/AsTeRICS-Grid/assets/229352/61db1684-0f59-48fe-b4d0-94569f021619">
<img width="1375" alt="Screenshot 2023-09-12 at 10 35 03" src="https://github.com/asterics/AsTeRICS-Grid/assets/229352/e8edae08-00da-4374-b9c1-a53178336553">

**Comment by willwade** [comment]
NB: I wonder if this is all part of #168

**Comment by klues** [comment]
Currently you cannot insert line breaks into labels. I haven't seen this idea before, so I also don't think it's currently planned for #168.

However, of course we can think about it and maybe implement it at some time, if it makes sense for some use cases. I'm not quite sure what your use case is. How are you planning to use your book with an AAC user? If it's about interacting with the story, maybe it makes sense to create a separate element for each sentence?! Or if it's about "reading" the book, maybe you could just insert a picture without text with an background action to speak custom text which is the text of the current page?!
Why do you think line breaks within an element would be useful?

**Comment by willwade** [comment]
The book is both an example and template of a real communication book for an adult - non-literate - user

We want to publish it as a free resource for therapists and families as a guide. We will create a case study to go alongside it e.g 

> Sue is an adult with Multiple Sclerosis. She can understand some whole words and do some inital letters of words - but largely benefits from sone phrases in a book. She has a dog called Pepper, a husband called Bill and loves goinhg on holiday in her caravan. This is an example of the types of language that benefits Sue. Think of the person you are supporting and use it as a guide to think of relevant phrases and words

As much as the book is standalone - a therapist can use it as inspiration - it would make sense for therapists to edit it. That front page is just a big block of text. It could be in a different style (it was made in communicate in Print) - but multi-line text is the key on that page. The workaround as far as I can tell is pop it in a image - but that makes the editing not so easy to do.  So to clarify - its for communication partners. It would be a printed book - not an online communication tool. (well, not primarily).

**Comment by willwade** [comment]
Yeah - granted - I don't think its a top priority. I do think the only use cases is when the primary target of grid is for a pdf.. 

On-screen its all fine.
